sections/exchange/FooterCard/NoSynthsCard/NoSynthsCard.tsx

```diff
@@ -40,16 +40,28 @@ const NoSynthsCard: FC<NoSynthsCardProps> = ({ attached }) => {
 					</Message>
 				</DesktopOnlyView>
 				{isL2 ? (
-					<Link href={'https://staking.synthetix.io/staking'}>
-						<MessageButton size="lg" variant="primary" isRounded={true}>
-							<Trans
-								t={t}
-								i18nKey="exchange.onboard.mint-button"
-								values={{ currencyKey: sUSD }}
-								components={[<NoTextTransform />]}
-							/>
-						</MessageButton>
-					</Link>
+					<>
+						<a href="https://app.1inch.io/#/1/swap/USDC/sUSD" target="_blank" rel="noreferrer">
```

Comment:
I would suggest to avoid hardcoded `<a>` links and use `ExternalLink` for styling (it already includes the target and rel props), and add the URLs from `constants/links` for consistency;  it will make it easier to change them throughout the dApp in the future. See for reference how it is done at [GetL2GasCard](https://github.com/kwenta/kwenta/blob/fe65763fd840db34622dad9a38750a6c76d2bdbd/sections/exchange/FooterCard/GetL2GasCard/GetL2GasCard.tsx).

Reply:
Actually, Jeremy suggested we change the specs here as to just show one button with the link to 1inch, given that 1inch already routes to Uniswap